.github/workflows/docker_image.yaml

```
@@ -0,0 +1,39 @@
+name: Build and publish Docker Image
+
+on:
+  push:
+    tags:
+    - "*"
+
+jobs:
+  multi:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
```

I would prefer if we use the formatting we otherwise have as well.

```suggestion
      - name: Checkout
        uses: actions/checkout@v2
```